### ISSUE TITLE
Loadouts spawns player with incorrect ammo.

### ISSUE THREAD

**Issue by ghost** [Weol/WUMA#6]
Hi there,

I've found that if I go into the WUMA menu and set up a default loadout for myself (superadmin), I can edit the ammo amounts and they will change respectively when I do. However, upon respawning, I either have no ammo in the weapon or some amount that was not what I had entered. For example, with m9k weapons, I will have a full clip and some arbitrary amount that I'm assuming m9k gives on weapon spawn as a default. But if I use the basic HL2 weapons in my loadout, I'll spawn with just a full clip and nothing else (e.g. nine in pistol, six in shotgun, etc.)

To recap: If I enter an ammo amount in the loadouts menu, I will get that amount, but not spawn with it the next time I die or login.

So far I've enjoyed WUMA very much; it serves as a great tool for somebody like me who knows nothing about administrating a server but still needs to keep his friends in line. So thanks, and I hope this can get fixed soon.

**Comment by Weol** [comment]
Thank you for telling me, I will look into this. There may be a conflict with another addon, can you list all the addons you are running so I can try to replicate this issue?

**Comment by ghost** [comment]
Oof... that's where some work comes in. I've got something near 26 pages of mods I'm subscribed to. However, I do keep them all in collections to make it easier on my friends to download them. Do you think I could possibly just provide you with links to my six collections instead? (I categorize my mods into six types)

Edit: Not to mention that 26 pages worth of mods might be a bit much to ask...

**Comment by Weol** [comment]
Nevermind, I found the bug. It is a small fix, I'll push it soon. It may take up couple of days before it is live on the workshop, however.

**Comment by Weol** [comment]
Fixed by **ac8c90d2f0e57fd17043adb2a65b3c76024d37ee**
